include/zephyr/usb/usb_device.h

```diff
@@ -79,10 +79,10 @@ extern "C" {
  *  USB configuration
  **************************************************************************/
 
-#define USB_MAX_CTRL_MPS	64   /**< maximum packet size (MPS) for EP 0 */
-#define USB_MAX_FS_BULK_MPS	64   /**< full speed MPS for bulk EP */
-#define USB_MAX_FS_INT_MPS	64   /**< full speed MPS for interrupt EP */
-#define USB_MAX_FS_ISO_MPS	1023 /**< full speed MPS for isochronous EP */
+#define USB_MAX_CTRL_MPS	CONFIG_USB_DC_EP0_MPS   /**< maximum packet size (MPS) for EP 0 */
```

Comment:
Same as for #56977.
This is USB device stack header. I will not accept dependencies on any DT macros or Kconfig options in this header. I was probably not clear enough in https://github.com/zephyrproject-rtos/zephyr/pull/52278#issuecomment-1401895950, I apologize. To be clear (also fro https://github.com/zephyrproject-rtos/zephyr/pull/56838), the correct way would be to let the stack get control MPS information using the usb_dc driver API, see https://github.com/zephyrproject-rtos/zephyr/pull/57028

Reply:
OK, understood. I applied locally https://github.com/zephyrproject-rtos/zephyr/pull/57028 with a little modification of usb_dc driver and it works perfectly. Thanks! Will wait for merging this one (https://github.com/zephyrproject-rtos/zephyr/pull/57028) and then do the rest changes on Telink fork. Will close this PR since it's not needed anymore.